src/main/resources/jenkins/branch/PropertyMigration/MonitorImpl/description.properties

```diff
@@ -0,0 +1,24 @@
+#
+# The MIT License
+#
+# Copyright (c) 2018, CloudBees, Inc., Stephen Connolly.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+blurb=Allows administrators to migrate folder properties.
```

Comment:
Should probably clarify that this is for fixing deprecated formats. Example: https://github.com/jenkinsci/branch-api-plugin/blob/a9188b5ba942fd6d0f649ac40842726c6ceb0ff9/src/main/resources/jenkins/branch/Messages.properties#L58-L63

Reply:
[Like so?](https://github.com/jenkinsci/branch-api-plugin/pull/471/commits/d0b270195588bec2ed787e320030450c5fac80fc)

Reply:
Yeah seems good enough, thanks!